.changeset/pink-countries-pump.md

```diff
@@ -0,0 +1,16 @@
+---
+'@backstage/plugin-catalog': minor
```

Comment:
At stability index 1 with this one, so let's provide a bit more guidance around how to migrate existing custom catalog pages, as much as is sensible anyway

Reply:
Still need to update this, will take a look tomorrow.